Makefile

```diff
@@ -80,9 +84,8 @@ check_llvm_config = $(eval \
 .PHONY: all
 all: crystal ## Build all files (currently crystal only) [default]
 
-.PHONY: spec
-spec: $(O)/all_spec ## Run all specs
-	$(O)/all_spec $(SPEC_FLAGS)
+.PHONY: test ## Run tests
```

Comment:
For `make help` the `##` should go on the same line as the target itself, not the phony one (alternatively `test:` can be declared on an additional line like what `smoke_test` does)